src/Core/Enums/PaymentMethodType.cs

```diff
@@ -17,8 +17,10 @@ public enum PaymentMethodType : byte
     [Display(Name = "Wire Transfer")]
     WireTransfer = 5,
     [Display(Name = "Apple In-App Purchase")]
+    [Obsolete("In-app purchase payment method is not allowed.")]
     AppleInApp = 6,
     [Display(Name = "Google In-App Purchase")]
+    [Obsolete("In-app purchase payment method is not allowed.")]
```

Comment:
❓ Same -- do we feel OK with just removing these? Then I can remove the logic checks too, if we believe this type will never come in.